website/static/css/pages/login-page.css

```diff
@@ -38,3 +38,12 @@
     background-color: #CCC;
     border-color: #BBB;
 }
+.existing-user-signin {
+    /*height: 320px;*/
+    background-color: #EEE;
+    border-radius: 4px;
+    margin-top: 75px;
+}
+.existing-user-signin-button {
+    margin-right: 0px !important;
```

Comment:
Again, we should really avoid doing manual spacing like this and let Bootstrap do the job. Otherwise you run into responsiveness problems like this:

![osf___sign_in](https://cloud.githubusercontent.com/assets/2379650/15757982/53a92902-28d6-11e6-8e36-1cad959227d8.png)